.gitignore

```diff
@@ -0,0 +1 @@
+vendor
```

Comment:
> I've seen so many permutations of this, vendor, /vendor, /vendor/, /vendor/* and so on. Happy to change it, but do you happen to know a reasoning behind adding the trailing slash? Just to specify that it's a directory?

True, there have been so many permutations. In opinion, yes because it is a directory. It was just a suggestion, I won't be offended either way :smile: 

> `composer.phar` doesn't exist in the repo?

True. But so many people do have it, I found it is a good to have it in the ignore list, because someone will eventually commit it. Just my 2 cents :wink: 

@jmauerhan thoughts?